README.md

```diff
@@ -1,19 +1,46 @@
 # toffy
-Scripts for interacting with and generating data from the commercial MIBIScope
+Scripts for interacting with and generating data from the commercial MIBIScope.
+
+### System
+Depending on what operating system you are planning to install toffy on, the requirements will differ slightly.
+- [Install on Windows](#windows)
+- [Install on macOS](#macos)
 
 ## Requirements
+### Windows
 
-Latest version of conda (miniconda prefered)
+- You must have **C++ Build Tools** (VS19) installed. 
+Go to  https://visualstudio.microsoft.com/visual-cpp-build-tools/ and click 'Download Build Tools'.
+Open the installer and make sure you are installing the package labeled *C++ build tools*, then follow the prompts.
+    - **(If installing on CAC, you will need the admin password and must contact support@ionpath.com)**
 
-## Setup
+- You will need the latest version of Anaconda (**Miniconda** preferred). 
+Download here: https://docs.conda.io/en/latest/miniconda.html and select the appropriate download for your system.
+Choose "Just Me" option for installation, and do not need to select the "Tutorial" or "Getting Started" options.
+Continue with the installation.
 
-Clone the repo
+### macOS
+- You will need the latest version of Anaconda (**Miniconda** preferred). 
+Download here: https://docs.conda.io/en/latest/miniconda.html and select the appropriate download for your system.
+Choose "Just Me" option for installation, and do not need to select the "Tutorial" or "Getting Started" options.
+Continue with the installation.
+
+## Setup
+* For Windows, you will need open the Anaconda powershell prompt instead of the regular powershell prompt for the following.
+![](C:\Users\Cameron Sower\Desktop\Angelo Lab\toffy\templates\img\conda_powershell.png)
```

Comment:
This img will need to be added to toffy in order to show up

Reply:
dang I forgot to switch the path name thanks